packages/salesforcedx-utils-vscode/src/test/testrunner.ts

```diff
@@ -78,7 +78,6 @@ function run(testsRoot: any, clb: any): any {
     try {
       // Fill into Mocha
       files.forEach((f): Mocha => {
-        console.log(`Adding test script: ${f}`);
```

Comment:
Remove logging that @guw added. I don't think we need these and there were cluttering up the results when running locally.